app/code/core/Mage/Core/Model/Config.php

```diff
@@ -1331,21 +1306,51 @@ public function getPathVars($args = null)
         return $path;
     }
 
+    /**
+     * Retrieve class name from config.xml node
+     *
+     * @return string|false
+     */
+    public function getNodeClassName(string $path)
+    {
+        $config = Mage::getConfig()->getNode($path);
+        if (!$config) {
+            return false;
+        }
+        return $config->getClassName();
+    }
+
+    /**
+     * Retrieve class instance from config.xml node
+     *
+     * @param string $path
+     * @return Mage_Core_Model_Abstract|false
+     */
+    public function getNodeClassInstance($path)
```

Comment:
```suggestion
    /**
     * Retrieve class instance from config.xml node
     *
     * @return Mage_Core_Model_Abstract|false
     */
    public function getNodeClassInstance(string $path)
```

Reply:
This is actually not a new method. I moved it up a bit in the class definition since it wasn't relevant to the other code surrounding it.

Reply:
Mhh, should not do reviews online. Thanks

Reply:
HAHA. i did not knew it :) Why isnt this default?

> I debated not moving the methods at all to make review easier, but the random ordering put in by Mage devs just made me mad..

Yep. There are rules for phpcs-fixer that re-order methods .... coming next year ;)